releasenotes/notes/pauli-evo-plugins-612850146c3f7d49.yaml

```diff
@@ -0,0 +1,32 @@
+---
+features_quantum_info:
+  - |
+    Added :meth:`.SparsePauliOperator.to_sparse_list` to convert an operator into
+    a sparse list format. This works analogous to :meth:`.SparsePauliOperator.from_sparse_list`.
```

Comment:
Shouldn't this be "oppositely" rather than "analogously"?

Reply:
yeah maybe 😛 I'll update

Reply:
done in cf7d8b096